app/views/users/index.html.erb

```diff
@@ -13,11 +13,9 @@
 
 <% @users.each do |user| %>
   <tr>
-    <td><%= user.name %></td>
+    <td><%= link_to user.name, user_path(user) %></td>
     <td><%= user.email %></td>
-    <td><%= link_to 'Show', user_path(user) %></td>
-    <td><%= link_to 'Edit', edit_user_path(user) %></td>
-    <td><%= link_to 'Impersonate', impersonate_user_path(user), method: 'post' %></td>
+    <td><%= link_to 'Impersonate', impersonate_user_path(user), method: 'post', class: "btn btn-danger" %></td>
```

Comment:
impersonate should be angry! :) it is dangerous. we want to use the color to warn the admin before impersonate.